examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p>May be rephrase <code>The server is installed in the merlin-tensorflow container.</code> Since now we have a single container. Also, not all inference containers can be used here, TF backend is required. so let's specify as <code>merlin-tensorflow</code> .</p><p> </p><p>May be rephrase <code>specify the same path as the export_path ... </code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I copied that from the merlin system example - we need to update that too